Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -913,6 +914,9 @@ WebPageProxy::WebPageProxy(PageClient& pageClient, WebProcessProxy& process, Ref
     if (RefPtr gpuProcess = GPUProcessProxy::singletonIfCreated())
         gpuProcess->setPresentingApplicationAuditToken(process.coreProcessIdentifier(), m_webPageID, m_presentingApplicationAuditToken);
 #endif
+
+    Ref aboutSchemeHandler = AboutSchemeHandler::singleton();
```

Comment:
We shouldn't need `Ref` here given that this is a singleton. The static analyzer should be able to recognize this. I don't even think we need a local variable here.